integration/test_collection.py

```diff
@@ -1615,7 +1610,7 @@ def test_optional_ref_returns(collection_factory: CollectionFactory) -> None:
         references=[
             ReferenceProperty(name="ref", target_collection=ref_collection.name),
         ],
-        vectorizer_config=Configure.Vectorizer.none(),
+        vector_config=Configure.Vectors.none(),
```

Comment:
What is the rationale for renaming/changing the abstraction? I understand we are aiming to move all vector config behind the term "Vector" as opposed to having a combination of index type + vectorizer + extra config.

If so, I think it's a **good change**. The three are closely related and I find that it doesn't add too much value to think of them separately.

Reply:
The idea is to deprecate and then remove the `Configure.Vectorizer` and `Configure.NamedVectors` factories replacing them with `Configure.Vectors`. This necessarily requires `vectorizer_config` to be deprecated as a field and replaced with `vector_config` that has a different type signature. This way, we can bring the Python client more inline with the TS and future client rewrites